internal/k8s/controller.go

```diff
@@ -190,6 +190,29 @@ func AddRoutesFromNSToIngestionQueue(numWorkers uint32, c *AviController, namesp
 		utils.AviLog.Debugf("key: %s, msg: %s for namespace: %s", key, msg, namespace)
 	}
 
+}
+func AddServicesFromNSToIngestionQueue(numWorkers uint32, c *AviController, namespace string, msg string) {
+	// For Advancd L4 and service api , do not handle. services already been taken care
+	// in service handler
+	if lib.GetAdvancedL4() || lib.UseServicesAPI() {
```

Comment:
I think we should handle it for the ServicesAPI in the next PR.

Reply:
Will Handle ServiceAPI in next PR